### DISCUSSION TITLE
Question about a feature in previous version

### DISCUSSION THREAD

**Discussion by jhoanmartinez** [obsproject/obs-studio#10410] [General]
Guys, do you remember the version where you are able to pause the recording and also return the recording? I update the obs and the option is no longer available, thank you fellas.

**Comment by mihawk90** [comment]
That option is still available, but not when using the stream encoder.

**Reply by jhoanmartinez** [reply]
What should I check to fix it?

**Reply by mihawk90** [reply]
Your recording encoder options. It can't be set to "same as stream".

**Reply by jhoanmartinez** [reply]
Huge thanks, it worked

**Comment by jhoanmartinez** [comment]
@mihawk90 was right, it was a set up issue, not a version issue

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance in the future.

Thank you!
